presto-testing-docker/src/main/java/io/prestosql/testing/docker/DockerContainer.java

```diff
@@ -237,6 +242,27 @@ private void removeContainer(String containerId)
         }
     }
 
+    public DockerContainer execute(String... arguments)
+    {
+        try {
+            ImmutableList<String> command = ImmutableList.copyOf(arguments);
+            ExecCreation execCreation = dockerClient.execCreate(containerId, arguments, attachStdout(), attachStderr());
+            dockerClient.execStart(execCreation.id());
+            RetryPolicy<Object> retryPolicy = new RetryPolicy<>()
+                    .withMaxDuration(Duration.of(10, MINUTES))
+                    .withMaxAttempts(Integer.MAX_VALUE)
+                    .withBackoff(100, 2000, MILLIS)
+                    .onRetry(event -> LOG.info(format("Waiting for command: '%s' [%s]...", command, event.getLastFailure())));
+            Failsafe.with(retryPolicy).run(() -> checkState(!dockerClient.execInspect(execCreation.id()).running(), "Command is still running"));
+            ExecState execState = dockerClient.execInspect(execCreation.id());
+            checkState(execState.exitCode() == 0, "Command '%s' exited with: %s", command, execState.exitCode());
```

Comment:
here and above: s/`'%s'`/`%s
-- since you're using List.toString, the apostrophes will look funny around this